src/lib/components/Hamburger.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import HamburgerIcon from "$lib/images/hamburger-icon.svg"
+  import CloseIcon from "$lib/images/close-icon.svg"
+
+  export let popover: any
+  export let showInTablet: boolean = false
+  export let alignIconToRight: boolean = false
+</script>
+
+<button
+  use:popover.button
+  class={`hamburger-button ${showInTablet ? "tablet" : ""} ${
+    alignIconToRight ? "align-right" : ""
+  }`}
+>
+  {#if !$popover.expanded}
+    <HamburgerIcon />
+  {:else}
+    <CloseIcon />
+  {/if}
+</button>
+
+<style lang="scss">
+  @import "$styles/setup";
+
+  .hamburger-button {
+    display: none;
+    padding: 0;
+    background-color: transparent;
+    border: none;
+
+    @include mobile() {
+      display: block;
+    }
+
+    &.tablet {
+      @include tablet() {
+        display: block;
+      }
+    }
+
+    &.align-right {
+      float: right;
+    }
```

Comment:
I'd prefer basically any other way of aligning this than `float`, it tends to cause a lot of issues